### PR TITLE
Makefile shortcuts, and SCSS compilation pipeline

### PR DESCRIPTION
**Pull request by Mirzero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,3 +74,22 @@ The drupal Backup & Migrate module does not currently support PostgresQL. Backin
...
+	1. up 
+	1. down
+	1. start
+	1. stop
+	1. prune
+	1. ps
+	1. shell
+	1. drush <command>
+	1. logs <container>
+  1. compilescss
```
Comment (infojunkie) [inline README.md:93]: identation issue

Comment (Mirzero) [reply README.md]: Fixed and pushed.

File: src/.gitignore [hunk 2]
```diff
@@ -28,3 +28,6 @@
 # Ignore sass cache files
 .sass-cache/
 *.scssc
+
+# ignore node-modules, they should be redownloaded
+node_modules/**
```
Comment (infojunkie) [inline src/.gitignore:33]: remove `**` to ignore the whole folder

File: src/web/themes/custom/workbc/README.md [hunk 3]
```diff
@@ -9,12 +8,9 @@ See https://developer.gov.bc.ca/Typography for more details.
...
-install [sass](https://sass-lang.com/install).
-To compile, run from subtheme directory: `sass scss/style.scss css/style.css`
-
-If you get an error looking like `error scss/style.scss (Line 55 of <rootpath>/src/web/themes/contrib/bootstrap5/dist/bootstrap/5.1.3/scss/mixins/_utilities.scss: Invalid CSS after "...ass}: #{$value}": expected "{", was ";")`
-it means your sass compiler is out of date. 
-If you happen to be compiling through Ruby Sass, a `gem update sass` should solve the issue, otherwise update your sass installation.
-`sass -version` should tell you more.
-
+The php Docker container has Yarn, Grunt, and grunt-dart-sass set up for compilation.
+[grunt dart-sass] will compile everything starting with the style.scss, and
```
Comment (infojunkie) [inline src/web/themes/custom/workbc/README.md:12]: missing URLs

File: src/Makefile [hunk 4]
```diff
@@ -49,40 +45,43 @@ stop:
...
 ##		logs php	: View `php` container logs.
 ##		logs nginx php	: View `nginx` and `php` containers logs.
 .PHONY: logs
 logs:
-	@docker-compose logs -f $(filter-out $@,$(MAKECMDGOALS))
+	@docker-compose --project-directory "$(SOLUTION_ROOT)" logs -f $(filter-out $@,$(MAKECMDGOALS))
+
+## compilescss	:	Compile the scss per Gruntfile configuration
+.PHONY: compilescss
+compilescss:
```
Comment (infojunkie) [inline src/Makefile:77]: I ran this and got `exec: "grunt": executable file not found in $PATH`. I also tried after `yarn install` and got the same result, even though I can see the `grunt` executable inside `node_modules/.bin`. Needs fixing and documentation on installing the dependencies.

**Review by infojunkie — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**